src/lib.rs

```diff
@@ -67,10 +68,33 @@ pub struct RenderPass {
     texture_version: Option<u64>,
     next_user_texture_id: u64,
     pending_user_textures: Vec<(u64, egui::Texture)>,
-    user_textures: Vec<Option<wgpu::BindGroup>>,
+    //for support changing texture
+    user_textures: Vec<Option<(Arc<wgpu::Texture>,wgpu::BindGroup)>>,
```

Comment:
Why do we have to Arc the texture? Aren't we supposed to be the owner?

Reply:
wgpu::Texture does not implement copy or clone and skip work for life time

Reply:
But why don't we just keep the ownership of the texture? Does the application still need a reference for it?